docs/release_process.rst

```diff
@@ -248,3 +248,68 @@ Wrap-up
 * `Close the milestone <https://github.com/learningequality/kolibri/milestones>`__ on Github
 * For issues on this milestone that have been reported by the community, try to report in appropriate forum threads that the new release addresses the issues
+
+
+Send upgrade notifications
+--------------------------
+
+Wait about 3 business days after communications are published to see if any issues are reported. Afterwards, we can send upgrade notifications through the Nutrition Facts telemetry server.
+
+* `Log in to the telemetry server <https://telemetry.learningequality.org/account/login/google-oauth2/?next=/admin/>`__ using your Learning Equality Google Apps account
+* Create a new Message object by clicking the "+ Add" button
+* In the "Status" dropdown, select the "Staged" option
+* Set the link URL to ``https://learningequality.org/r/upgrade_kolibri``
+* In the "Version range" field, enter a valid semver range (e.g. >=0.12.0)
+* Generate and add a new internationalized ``i18n`` JSON blob using the ``nutritionfacts_i18n.py`` script as shown below:
+
+.. code-block:: bash
+
+  python build_tools/i18n/nutritionfacts_i18n.py
+
+You can also specify specific string IDs for the ``title``, ``msg``, and ``link_text``, e.g.:
+
+.. code-block:: bash
+
+  python build_tools/i18n/nutritionfacts_i18n.py --message UpdateNotification.upgradeMessage0124
+
+This will output a JSON blob like:
+
+.. code-block:: text
+
+  {
+    "ar": {
+      "link_text": "تعلم المزيد وقم بتحميله هنا",
+      "msg": "هناك إصدار جديد متاح من كوليبري.",
+      "title": "التحديث للنسخة الجديدة أصبح متاحاً"
+    },
+    "bg-bg": {
+      "link_text": "Научи повече и изтегли оттук",
+      "msg": "Налична е нова версия на Колибри.",
+      "title": "Има налични подобрения"
+    },
+    "bn-bd": {
+      "link_text": "আরও জানুন এবং সেটি এখানে ডাউনলোড করুন",
+      "msg": "কলিব্রির একটি নতুন সংস্করণ পাওয়া যাচ্ছে।",
+      "title": "আপগ্রেড উপলব্ধ"
+    },
+    "en": {
+      "link_text": "Learn more and download it here",
+      "msg": "A new version of Kolibri is available.",
+      "title": "Upgrade available"
+    },
+    "es-es": {
+      "link_text": "Descubre más y descarga aquí",
+      "msg": "Una nueva versión de Kolibri está disponible.",
+      "title": "Actualización disponible"
+    },
+    //...
+  }
+
+
+You can `redirect this output to a file <https://askubuntu.com/questions/420981/how-do-i-save-terminal-output-to-a-file>`_ (Bash) or `pipe it to the clipboard <https://stackoverflow.com/questions/1753110/>`_ (Mac)
+
+Set Kolibri's ``RUN_MODE`` to ``staged-msgs`` to receive staged messages. Test that all languages are displayed correctly.
+
+Next, emulate different versions and ensure that the semver conditional logic is being processed correctly. Set ``RUN_MODE`` to something like ``staged-msgs-ver-0.12.3`` to emulate version 0.12.3, for example.
```

Comment:
How does this emulate a different version? Does it parse the `RUN_MODE` to infer the version?

Reply:
That's how nutrition facts seems to work. Maybe could clarify that by saying that in general set KOLIBRI_RUN_MODE to `staged-msgs-ver-<semver version number>`